.github/workflows/ci.yml

```diff
@@ -208,6 +208,17 @@ jobs:
             source plugin/trino-hive-hadoop2/conf/hive-tests-${{ matrix.config }}.sh &&
               plugin/trino-hive-hadoop2/bin/run_hive_s3_tests.sh
           fi
+      - name: Run Hive S3 Select JSON Tests
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESSKEY }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRETKEY }}
+          S3_BUCKET: "presto-ci-test"
+          S3_BUCKET_ENDPOINT: "s3.us-east-2.amazonaws.com"
+        run: |
+          if [ "${AWS_ACCESS_KEY_ID}" != "" ]; then
+            source plugin/trino-hive-hadoop2/conf/hive-tests-config-hdp3.sh &&
```

Comment:
Shouldn't this step be skipped if the matrix config is not hdp3?

Reply:
Good point, I'll add this check.

Reply:
Hi @nineinchnick I've added the check and all tests are passing now. Can you please take another look?